recipes/qpoases/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package CXX)
+
+find_package(qpOASES REQUIRED)
```

Comment:
```suggestion
project(test_package CXX)

include("${CMAKE_BINARY_DIR}/conanbuild.info.cmake")
conan_basic_setup(TARGETS)

find_package(qpOASES REQUIRED)
```

Reply:
Does this mean this lib cannot be built on windows without using conan specific functionality inside CMakeLists.txt?

Reply:
It can be built.
You can manually pass the correct flags to the compiler.
Conan has these helpers to automatically set the MSVC runtime

Reply:
To know more about static vs shared runtime,
I suggest reading the msdn documentation
https://docs.microsoft.com/en-us/cpp/build/reference/md-mt-ld-use-run-time-library?view=msvc-160